orangecanvas/application/addons.py

```diff
@@ -69,9 +69,9 @@ def prettify_name(name):
     dash_split = name.split('-')
     # Orange3-ImageAnalytics => ImageAnalytics
     if len(dash_split) > 1 and dash_split[0].lower() in ['orange', 'orange3']:
-        name = '-'.join(dash_split[1:])
+        name = ' '.join(dash_split[1:])
     # ImageAnalytics => Image Analytics  # while keeping acronyms
-    return re.sub(r"(?<!^)([A-Z][a-z]|(?<=[a-z])[A-Z])", r" \1", name)
+    return re.sub(r"(?<!^)([A-Z][a-z]\s\s+|(?<=[a-z])[A-Z])", r" \1", name)
```

Comment:
Although this works, I don't really understand it. Why `\s\s+`? As far as I understand, this now only has to insert spaces into camel case names. Wouldn't a simpler `r"(?<!^)(?<! )([A-Z])"` or even `r"(?<=[a-z])([A-Z])"` work as well?